packages/e2e-tests/tests/export.spec.ts

```diff
@@ -48,10 +48,9 @@ test.describe("Export tests", () => {
       const dialog = app.mainWin.getByRole("dialog")
       await dialog.getByRole("radio", {name: `${label}`}).click()
       await dialog.getByRole("button").filter({hasText: "Export"}).click()
-
-      await expect(
-        await app.mainWin.locator("text=Export Complete").first()
-      ).toBeVisible()
+      await app.mainWin
+        .getByText(new RegExp("Export Completed: .*results." + label))
```

Comment:
I'll point out that the final `.` here should ideally be a `\.` since we're matching against the separator before the file extension. However, when I had `\.` in there, the linter grumped about it being an "Unnecessary escape character". Per [this post](https://stackoverflow.com/questions/47277133/disable-unnecessary-escape-character-no-useless-escape), I'm not the only person to have bumped into this. I had it working with the `//eslint-disable-line` to shut it up, but since the regexp is still specific enough to get the job done even with the `.` matching "any character" I left it like this. Happy to switch it over if folks feel otherwise.

Reply:
I think the linter's complaint is legitimate. You want `Regexp()` to see a string that ends with `\.`, and because `\` is the string escape character, you need to escape the `\` itself.
```suggestion
        .getByText(new RegExp("Export Completed: .*results\\." + label))
```

Reply:
Thanks @nwt. Fixed!